tests/TestCase/Core/ConfigureTest.php

```diff
@@ -464,8 +464,14 @@ public function testStoreAndRestoreWithData()
      */
     public function testVersion()
     {
-        $result = Configure::version();
-        $this->assertTrue(version_compare($result, '4.0', '>='));
+        $original = Configure::version();
+        $this->assertTrue(version_compare($original, '4.0', '>='));
+
+        Configure::write('Cake.version', 'banana');
+        $this->assertSame('banana', Configure::version('Cake.version'));
+
+        Configure::delete('Cake.version');
```

Comment:
So, if you delete Cake.version or don't have it in your config, it will force overwrite your config with config/config.php values?

Reply:
Kind of, you could define `Cake.version` to skip the need to load the file. However if the value is undefined/null the config/config.php file will be loaded.